src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1171,6 +1179,9 @@ bool ReconstructionEngine_sequentialSfM::getBestInitialImagePairs(std::vector<Pa
     ALICEVISION_LOG_WARNING("Failed to find an initial pair automatically. There is no view with valid intrinsics.");
     return false;
   }
+
+  if(filterViewId != UndefinedIndexT)
+    ALICEVISION_LOG_INFO("Selection of an initial pair with one given view id.");
```

Comment:
`ALICEVISION_LOG_INFO("Selection of an initial pair with one given view id: " << filterViewId);`